rest_api/VulnMgmtService.adoc

```diff
@@ -0,0 +1,3129 @@
+
+
+
+
+
+
+[.fields-StorageImageScanNote]
+[cols="1"]
+|===
+| Enum Values
+
+| UNSET
+| OS_UNAVAILABLE
+| PARTIAL_SCAN_DATA
+| OS_CVES_UNAVAILABLE
+| OS_CVES_STALE
+| LANGUAGE_CVES_UNAVAILABLE
+| CERTIFIED_RHEL_SCAN_UNAVAILABLE
+
+|===
+
+
+[#StorageImageSignature]
+=== _StorageImageSignature_ 
+
+
+
+
+[.fields-StorageImageSignature]
+[cols="2,1,1,2,4,1"]
+|===
+| Field Name| Required| Nullable | Type| Description | Format
+
+| signatures
+| 
+| 
+|   List   of <<StorageSignature>>
+| 
+|     
+
+| fetched
+| 
+| 
+|   Date  
+| 
+| date-time    
+
+|===
+
+
+
+[#StorageImageSignatureVerificationData]
+=== _StorageImageSignatureVerificationData_ 
+
+
+
+
+[.fields-StorageImageSignatureVerificationData]
+[cols="2,1,1,2,4,1"]
+|===
+| Field Name| Required| Nullable | Type| Description | Format
+
+| results
+| 
+| 
+|   List   of <<StorageImageSignatureVerificationResult>>
+| 
+|     
+
+|===
+
+
+
+[#StorageImageSignatureVerificationResult]
+=== _StorageImageSignatureVerificationResult_ Next Tag: 6
+
+
+
+
+[.fields-StorageImageSignatureVerificationResult]
+[cols="2,1,1,2,4,1"]
+|===
+| Field Name| Required| Nullable | Type| Description | Format
+
+| verificationTime
+| 
+| 
+|   Date  
+| 
+| date-time    
+
+| verifierId
+| 
+| 
+|   String  
+| verifier_id correlates to the ID of the signature integration used to verify the signature.
+|     
+
+| status
+| 
+| 
+|  <<StorageImageSignatureVerificationResultStatus>>  
+| 
+|    UNSET, VERIFIED, FAILED_VERIFICATION, INVALID_SIGNATURE_ALGO, CORRUPTED_SIGNATURE, GENERIC_ERROR,  
+
+| description
+| 
+| 
+|   String  
+| description is set in the case of an error with the specific error's message. Otherwise, this will not be set.
+|     
+
+| verifiedImageReferences
+| 
+| 
+|   List   of <<string>>
+| The full image names that are verified by this specific signature integration ID.
+|     
+
+|===
+
+
+
+[#StorageImageSignatureVerificationResultStatus]
+=== _StorageImageSignatureVerificationResultStatus_ 
+
+Status represents the status of the result.
+
+ - VERIFIED: VERIFIED is set when the signature's verification was successful.
+ - FAILED_VERIFICATION: FAILED_VERIFICATION is set when the signature's verification failed.
+ - INVALID_SIGNATURE_ALGO: INVALID_SIGNATURE_ALGO is set when the signature's algorithm is invalid and unsupported.
+ - CORRUPTED_SIGNATURE: CORRUPTED_SIGNATURE is set when the raw signature is corrupted, i.e. wrong base64 encoding.
+ - GENERIC_ERROR: GENERIC_ERROR is set when an error occurred during verification that cannot be associated with a specific
+status.
+
+
+
+
+[.fields-StorageImageSignatureVerificationResultStatus]
+[cols="1"]
+|===
+| Enum Values
+
+| UNSET
+| VERIFIED
+| FAILED_VERIFICATION
+| INVALID_SIGNATURE_ALGO
+| CORRUPTED_SIGNATURE
+| GENERIC_ERROR
+
+|===
+
+
+[#StorageLabelSelector]
+=== _StorageLabelSelector_ Label selector components are joined with logical AND, see     https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/
```

Comment:
Another potential parsing issue? There may be more, these just jumped out at me.
![Screenshot 2024-09-25 at 3 28 15 PM](https://github.com/user-attachments/assets/645fad6d-8cb7-4adb-91e7-d1b2d0664300)